### ISSUE TITLE
Readings for older Toon are incorrect

### ISSUE THREAD

**Issue by bbredewold** [cyberjunky/home-assistant-toon_smartmeter#44]
Hi,

It seems that the integration is a little off when working with my older (first edition) Toon.

<img width="426" alt="image" src="https://user-images.githubusercontent.com/2910910/147449238-9cf5c204-621c-4f29-b219-51df5e191245.png">

Maybe we can compare some of the raw outputs to fix this?

```json
{
	"dev_settings_device": {
		"uuid": "xxx",
		"name": "settings_device",
		"internalAddress": "settings_device",
		"type": "settings_device"
	},
	"dev_2": {
		"uuid": "xxx",
		"name": "HAE_METER_v2",
		"internalAddress": "2",
		"type": "HAE_METER_v2",
		"supportsCrc": "1",
		"ccList": "22 3c 3d 3e 56 60 70 72 7a 86 8b 73",
		"supportedCC": "22 3c 3d 3e 56 60 70 72 7a 86 8b 73",
		"nodeFlags": [],
		"IsConnected": "1",
		"HealthValue": "10",
		"DeviceName": "HAE_METER_v2",
		"CurrentSensorStatus": "UNKNOWN"
	},
	"dev_2.1": {
		"uuid": "xxx",
		"name": "HAE_METER_v2_1",
		"internalAddress": "2.1",
		"type": "HAE_METER_v2_1",
		"supportsCrc": "0",
		"ccList": "3c 3d 3e 72 86",
		"supportedCC": "3c 3d 3e 72 86",
		"nodeFlags": [],
		"CurrentGasFlow": "45.00",
		"CurrentGasQuantity": "1664155.00",
		"DeviceName": "HAE_METER_v2_1",
		"CurrentSensorStatus": "OPERATIONAL"
	},
	"dev_2.2": {
		"uuid": "xxx",
		"name": "HAE_METER_v2_2",
		"internalAddress": "2.2",
		"type": "HAE_METER_v2_2",
		"supportsCrc": "0",
		"ccList": "3c 3d 3e 72 86",
		"supportedCC": "3c 3d 3e 72 86",
		"nodeFlags": [],
		"DeviceName": "HAE_METER_v2_2",
		"CurrentElectricityFlow": "NaN",
		"CurrentElectricityQuantity": "NaN",
		"CurrentSensorStatus": "UNKNOWN"
	},
	"dev_2.3": {
		"uuid": "xxx",
		"name": "HAE_METER_v2_3",
		"internalAddress": "2.3",
		"type": "HAE_METER_v2_3",
		"supportsCrc": "0",
		"ccList": "3c 3d 3e 72 86",
		"supportedCC": "3c 3d 3e 72 86",
		"nodeFlags": [],
		"DeviceName": "HAE_METER_v2_3",
		"CurrentElectricityFlow": "220.00",
		"CurrentElectricityQuantity": "2861943.00",
		"CurrentSensorStatus": "OPERATIONAL"
	},
	"dev_2.4": {
		"uuid": "xxx",
		"name": "HAE_METER_v2_4",
		"internalAddress": "2.4",
		"type": "HAE_METER_v2_4",
		"supportsCrc": "0",
		"ccList": "3c 3d 3e 72 86",
		"supportedCC": "3c 3d 3e 72 86",
		"nodeFlags": [],
		"DeviceName": "HAE_METER_v2_4",
		"CurrentElectricityFlow": "0.00",
		"CurrentElectricityQuantity": "1169581.00",
		"CurrentSensorStatus": "UNKNOWN"
	},
	"dev_2.5": {
		"uuid": "xxx",
		"name": "HAE_METER_v2_5",
		"internalAddress": "2.5",
		"type": "HAE_METER_v2_5",
		"supportsCrc": "0",
		"ccList": "3c 3d 3e 72 86",
		"supportedCC": "3c 3d 3e 72 86",
		"nodeFlags": [],
		"DeviceName": "HAE_METER_v2_5",
		"CurrentElectricityFlow": "0.00",
		"CurrentElectricityQuantity": "2318511.00",
		"CurrentSensorStatus": "UNKNOWN"
	},
	"dev_2.6": {
		"uuid": "xxx",
		"name": "HAE_METER_v2_6",
		"internalAddress": "2.6",
		"type": "HAE_METER_v2_6",
		"supportsCrc": "0",
		"ccList": "3c 3d 3e 72 86",
		"supportedCC": "3c 3d 3e 72 86",
		"nodeFlags": [],
		"DeviceName": "HAE_METER_v2_6",
		"CurrentElectricityFlow": "0.00",
		"CurrentElectricityQuantity": "506376.00",
		"CurrentSensorStatus": "UNKNOWN"
	},
	"dev_4": {
		"uuid": "xxx",
		"name": "ZMNHTD1",
		"internalAddress": "4",
		"type": "ZMNHTD1",
		"supportsCrc": "1",
		"ccList": "5e 55 86 73 56 9f 72 5a 70 85 8e 59 32 6c 7a",
		"supportedCC": "5e 55 86 73 56 9f 72 5a 70 85 8e 59 32 6c 7a",
		"nodeFlags": [],
		"IsConnected": "1",
		"HealthValue": "10",
		"DeviceName": "ZMNHTD1",
		"CurrentElectricityFlow": "0.00",
		"CurrentElectricityQuantity": "500.00",
		"CurrentVoltage": "230.40",
		"CurrentCurrent": "0.23",
		"TargetStatus": "-1"
	},
	"dev_4.export": {
		"uuid": "xxx",
		"name": "ZMNHTD1_export",
		"internalAddress": "4.export",
		"type": "ZMNHTD1_export",
		"supportsCrc": "0",
		"nodeFlags": [],
		"CurrentElectricityQuantity": "20000.00",
		"CurrentElectricityFlow": "0.10",
		"DeviceName": "ZMNHTD1_export"
	}
}
```

**Comment by bbredewold** [comment]
It seems that connecting a modern Qubino smart meter to an old Toon creates this situation.

Created a pull request to fix this #45.

**Comment by ErikSiemens** [comment]
I’m not Shure its the same problem, but after updating HA yesterday to the latest version gas count is not working anymore

**Comment by cyberjunky** [comment]
> It seems that connecting a modern Qubino smart meter to an old Toon creates this situation.
> 
> Created a pull request to fix this #45.

I have committed it, it sometimes happen that all device ID's move when something is added.

**Comment by cyberjunky** [comment]
> I’m not Shure its the same problem, but after updating HA yesterday to the latest version gas count is not working anymore

@ErikSiemens can you gather some debug information to check it?
